commands/pipeline/run/pipeline_run.go

```diff
@@ -0,0 +1,97 @@
+package run
+
+import (
+	"fmt"
+
+	"github.com/profclems/glab/commands/cmdutils"
+	"github.com/profclems/glab/internal/git"
+	"github.com/profclems/glab/internal/utils"
+	"github.com/profclems/glab/pkg/api"
+
+	"github.com/MakeNowJust/heredoc"
+	"github.com/spf13/cobra"
+	"github.com/xanzy/go-gitlab"
+)
+
+func getDefaultBranch(f *cmdutils.Factory) (string) {
+	repo, err := f.BaseRepo()
+	if err != nil {
+		return "master"
+	}
+
+	remotes, err := f.Remotes()
+	if err != nil {
+		return "master"
+	}
+
+	repoRemote, err := remotes.FindByRepo(repo.RepoOwner(), repo.RepoName())
+	if err != nil {
+		return "master"
+	}
+
+	branch, _ := git.GetDefaultBranch(repoRemote.Name)
+
+	return branch
+}
+
+func NewCmdRun(f *cmdutils.Factory) *cobra.Command {
+	var pipelineRunCmd = &cobra.Command{
+		Use:   "run [flags]",
```

Comment:
Since the API uses 'create', we should add `create` as an alias to the run command with:
```go
.....,
Aliases: []string{"create"},
```